services/i18n.js

```diff
@@ -0,0 +1,53 @@
+const has = require('lodash/has');
+const get = require('lodash/get');
+
+const yaml = require('yamljs');
+const es = yaml.load('./locales/es.yml');
+const en = yaml.load('./locales/en.yml');
+
+const accepts = require('accepts');
+
+// default language
+let defaultLanguage = 'en';
+let language = defaultLanguage;
+const languages = ['en', 'es'];
+
+const translations = Object.assign(en, es);
+
+/**
+ * Exposes a service object to allow translations.
+ * @type {Object}
+ */
+const i18n = {
+
+  /**
+   * Create the new Task kue.
+   */
+  init(req) {
+    const lang = accepts(req).language(languages);
```

Comment:
This doesn't choose the client's preferred language, see: https://github.com/jshttp/accepts#languagelanguages. `accepts(req).languages()` should be used instead.